code/game/turfs/open/space/space.dm

```diff
@@ -144,7 +144,7 @@ GLOBAL_LIST_EMPTY(starlight)
 		return
 	if(istype(C, /obj/item/stack/rods))
 		build_with_rods(C, user)
-	else if(istype(C, /obj/item/stack/tile/iron))
+	else if(C.has_material_type(/datum/material/iron))
```

Comment:
```suggestion
	else if(istype(C, /obj/item/stack/tile) && C.has_material_type(/datum/material/iron))
```

My bad. We don't want any item made of iron but only tiles made of iron to be used.